### ISSUE TITLE
Module support issues in PowerShell 7 after installing SQL 2022

### ISSUE THREAD

**Issue by fmDBArchitect** [dataplat/dbatools#8673]
### Verified issue does not already exist?

I have searched and found no existing issue

### What error did you receive?

WARNING: [08:19:00][Invoke-DbaQuery] Failure | Could not load type 'Microsoft.SqlServer.Server.SqlContext' from assembly 'System.Data, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'.

### Steps to Reproduce

```powershell
# provide your command(s) executed pertaining to dbatools
# please include variable values (redacted or fake if needed) for reference
```
Invoke-DbaQuery -SqlInstance $sI -Database $tgtTB -Query "Select @@SERVERNAME AS [ServerName]";

Error is actually received on anything that tries to use SMO.
Tools still work in earlier versions of PowerShell, but cannot run scripts with Foreach-Object -parallel

### Please confirm that you are running the most recent version of dbatools

1.1.143

### Other details or mentions

This is likely a breaking change, not in DBAtools, but that Microsoft has moved some objects from SMO to Microsoft.SqlServer.SqlManagementObjects

### What PowerShell host was used when producing this error

PowerShell Core (pwsh.exe), VS Code (terminal), VS Code (integrated terminal)

### PowerShell Host Version

Name                           Value
----                           -----
PSVersion                      7.3.0
PSEdition                      Core
GitCommitId                    7.3.0
OS                             Microsoft Windows 10.0.22000
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0

### SQL Server Edition and Build number

Microsoft SQL Server 2022 (RTM) - 16.0.1000.6 (X64) 
	Oct  8 2022 05:58:25 
	Copyright (C) 2022 Microsoft Corporation
	Developer Edition (64-bit) on Windows 10 Enterprise 10.0 <X64> (Build 22000: )

### .NET Framework Version

.NET 7.0.0

PSChildName                      Version
-----------                      -------
v2.0.50727                       2.0.50727.4927
v3.0                             3.0.30729.4926
Windows Communication Foundation 3.0.4506.4926
Windows Presentation Foundation  3.0.6920.4902
v3.5                             3.5.30729.4926
Client                           4.8.04161
Full                             4.8.04161
Client                           4.0.0.0

**Comment by potatoqualitee** [comment]
Thank you for the report, @fmDBArchitect -- we've seen this too. Please try 5.1 until we release dbatools 2.0. We'll have a better idea of when that will be at the end of the week /cc @sanderstad

**Comment by potatoqualitee** [comment]
this will be fixed in 2.0

![image](https://user-images.githubusercontent.com/8278033/207067034-b09edcc1-0178-417e-afe1-2a2b3b9e26b6.png)

![image](https://user-images.githubusercontent.com/8278033/207067079-78a9a4be-abda-4fa2-ba87-b1a3389eaea9.png)

**Comment by simonsabin** [comment]
Any work around without going to 2.0?

We've machines that 2.0 won't work on.

**Comment by potatoqualitee** [comment]
hey simon, if it doesnt work on the latest 1.x version, then it wont work. but ultimately, there shouldn't be any machine that 2.0 won't work on.

**Comment by potatoqualitee** [comment]
Finally, this should be fixed in 2.5.5 for sure
